components/network-tx-button.tsx

```diff
@@ -105,8 +107,10 @@ function networkTxButton({
           }));
 
           if (answer.blockNumber)
-            processEvent("oracles","changed", state.Service?.network?.lastVisited, {fromBlock:answer.blockNumber})
-              .catch(console.debug);
+            processEvent("oracles",
```

Comment:
Like on the previous review, this component should not have such a specific event-processor call since this can be used app-wide and not just for lock/unlock